lib/galaxy/managers/group_roles.py

```diff
@@ -46,8 +39,8 @@ def show(self, trans: ProvidesAppContext, id: EncodedDatabaseIdField, group_id:
         group_role = self._get_group_role(trans, group, role)
         if group_role is None:
```

Comment:
```suggestion
        if not group_role:
```
Your version is perfectly fine. For a simple check that `foo` has a value, `if foo` is the simplest and cleanest option, and that's what we'll typically expect. Sometimes we need to check for equivalence with `None` explicitly (e.g. when a falsy value like the empty string, zero, empty list, etc. should evaluate to `True`, or when an object doesn't have the `__bool__` method implemented). But in all other cases, I would suggest sticking with the pythonic `if foo`.